main/services/devon-instances/devon-instances.service.ts

```diff
@@ -137,6 +157,64 @@ export default class DevonInstancesService implements SaveDetails {
     };
   }
 
+  getDevonIdeScriptsFromMaven(): Promise<DevonIdeScripts[]> {
+    let ideScripts: DevonIdeScripts[] = [];
+    let data = '';
+    const ideScriptsPromise = new Promise<DevonIdeScripts[]>(
+      (resolve, reject) => {
+        https
+          .get(
+            'https://search.maven.org/classic/solrsearch/select?q=g%3A%22com.devonfw.tools.ide%22%20AND%20a%3A%22devonfw-ide-scripts%22&rows=20&core=gav&wt=json',
+            (res) => {
+              res.on('data', (d) => {
+                data += d;
+              });
+              res.on('end', () => {
+                const jsonData = JSON.parse(data);
+                ideScripts = jsonData['response']['docs'].map((i) => {
+                  return { version: i.v, updated: i.timestamp };
+                });
+                resolve(ideScripts);
+              });
+            }
+          )
+          .on('error', (e) => {
+            reject('error: ' + e);
+          });
+      }
+    );
+    return ideScriptsPromise;
+  }
+
+  getLatestDevonIdeScriptsFromMaven(): Promise<DevonIdeScripts> {
+    let ideScript: DevonIdeScripts;
+    let data = '';
+    const ideScriptPromise = new Promise<DevonIdeScripts>((resolve, reject) => {
+      https
+        .get(
+          'https://search.maven.org/classic/solrsearch/select?q=a%3A%22devonfw-ide-scripts%22&rows=20&wt=json',
+          (res) => {
+            res.on('data', (d) => {
+              data += d;
+            });
+            res.on('end', () => {
+              const jsonData = JSON.parse(data);
+              const latestIdeScript = jsonData['response']['docs'][0];
+              ideScript = {
+                version: latestIdeScript.latestVersion,
+                updated: latestIdeScript.timestamp,
+              };
+              resolve(ideScript);
+            });
+          }
+        )
+        .on('error', (e) => {
+          reject('error: ' + e);
+        });
+    });
+    return ideScriptPromise;
+  }
+
```

Comment:
This  was moved to devonfw-ides.service.ts

getDevonIdeScriptsFromMaven --> is now --> getDevonfwIDEs
getLatestDevonIdeScriptsFromMaven --> is now --> getLatestDevonfwIDE

Check those methods and update them if needed